code/clue.js

```diff
@@ -53,7 +178,46 @@ const pickKiller = () => {
   ).innerHTML = `${mystery.killer.firstName} ${mystery.killer.lastName}`
 }
 
+document.getElementById('killerCard').addEventListener('click', pickKiller)
```

Comment:
After Vans lecture last Wednesday I moved all three event listeners to the end of the javascript file - seemed like a nice idea to have all execution lines of code gathered in one place.

Reply:
Yes, it's very clean approach